jgrapht-core/src/main/java/org/jgrapht/alg/interfaces/AllSimpleCyclesAlgorithm.java

```diff
@@ -0,0 +1,49 @@
+package org.jgrapht.alg.interfaces;
+
+import org.jgrapht.Graph;
+import org.jgrapht.GraphPath;
+
+import java.util.List;
+
+/**
+ * Created by svetl on 26.03.2017.
+ */
+public interface AllSimpleCyclesAlgorithm<V, E> {
+    /**
+     * Returns the graph on which the cycle base search algorithm is executed by this object.
+     *
+     * @return The graph.
+     */
+    Graph<V, E> getGraph();
```

Comment:
Remove this from the new interface. See the other interfaces in the same package as examples.